pkgs/by-name/or/oreo-cursors-plus/package.nix

```diff
@@ -0,0 +1,67 @@
+{
+  lib,
+  stdenv,
```

Comment:
Maybe you could try replacing `stdenv` with `stdenvNoCC`? I don't think this should require a C toolchain.